src/command.rs

```diff
@@ -111,7 +111,7 @@ pub enum Transaction {
 
         /// Name of the keypair. You can use existing keypair or generate new keypair with your preferred name using `./pchain_client keys create --name <NAME>`.
```

Comment:
The doc is still mentioning `--name` but not `--keypair-name`

Reply:
because for `./pchain_client keys create`, it is still using `--name`. It is `--keypair-name` only when we are submitting transaction. Should I make a change so that both of them use `--keypair-name` ?

Reply:
ya, please make both of them use `--keypair-name`.

Reply:
Resolved in commit [2960d96](https://github.com/parallelchain-io/pchain-client-cli/pull/9/commits/2960d961111eaed3d39013fa43edbb8c500006b8).